### ISSUE TITLE
Still plays 24bit files as 16 bit with (pulseaudio)

### ISSUE THREAD

**Issue by ckolivas** [clementine-player/Clementine#5944]
This issue was originally brought up here: https://github.com/clementine-player/clementine/issues/1747 but was not actually resolved. After it was closed, a separate patch placed into an out-of-mainline repository here https://github.com/izissise/Clementine/commit/9bc54924bf3ed775f3192e6d4a09eb0eb5ab0c66 was proposed to fix the issue. I have applied this patch myself to test it and it does indeed resolve the issue with 24 bit playback now working (and visualisations still working.) Please consider upstreaming this patch.

**Comment by santigl** [comment]
If I recall correctly, that was making the audio not work under Windows.

Are you experiencing that issue with the current master branch of the repo?

**Comment by ckolivas** [comment]
Yes it's still an issue on current master on linux and this patch referenced above fixes it.

**Comment by ckolivas** [comment]
Indeed, as an update, even the patch referenced doesn't fix it. Only removing the other caps fixed it.
